server/routes/trees/trees.test.js

```diff
@@ -40,12 +40,12 @@ describe('/api/trees/:id', () => {
         };
 
         const {
-          data: { idTree },
+          data: { id },
         } = await axiosAPIClient.post('/trees', body);
 
         /** Act */
         const params = {
-          currentTreeId: idTree,
+          currentTreeId: id,
```

Comment:
Has this test been run?  If I'm reading this correctly, I think the param name has been changed to `id`, so I wouldn't think this test would pass.